File: packages/shared/src/enums/confidence-label.ts
export const ConfidenceLabel = {
  New: "new",
  Learning: "learning",
  StrongMatch: "strong_match",
} as const;

export type ConfidenceLabel =
  (typeof ConfidenceLabel)[keyof typeof ConfidenceLabel];

export const CONFIDENCE_LABELS = Object.values(ConfidenceLabel);

File: LetsGo/LetsGo/Models/Enums/ConfidenceLabel.swift
import SwiftUI

enum ConfidenceLabel: String, Codable, Hashable {
    case strongMatch
    case learning
    case new

    var displayText: String {
        switch self {
        case .strongMatch: "Strong match"
        case .learning: "Learning"
        case .new: "New"
        }
    }

    var displayColor: Color {
        switch self {
        case .strongMatch: .strongMatchGreen
        case .learning: .learningOrange
        case .new: .newPurple
        }
    }
}
